examples/basic_voice_listening.py

```diff
@@ -0,0 +1,141 @@
+import discord
+import argparse
+
+
+def vc_required(func):
+    async def get_vc(self, msg):
```

Comment:
Examples are heavily scrutinized, and as such you do need type annotations and brief comments.

This decorated check should have a small piece of text explaining what it does.